src/plugins/dataflow/nodes/dataflow-node-plot.tsx

```diff
@@ -59,15 +95,18 @@ function lineData(node: any) {
       recentValues.forEach((val: any) => {
         if (isFinite(val)) {
           chData.push(val);
-          dsMax = Math.max(dsMax, val);
-          dsMin = Math.min(dsMin, val);
         }
       });
+      const localMax = Math.max(...chData);
+      node.data.dsMax = ((node.data.dsMax !== undefined) ? Math.max(localMax, node.data.dsMax) : localMax);
+      const localMin = Math.min(...chData);
+      node.data.dsMin = ((node.data.dsMin !== undefined) ? Math.min(localMin, node.data.dsMin) : localMin);
       dataset.data = chData;
       chartDataSets.push(dataset);
     }
   });
-  stepY = (dsMax - dsMin) / 2;
+
+  stepY = (node.data.dsMax  - node.data.dsMin) / 2;
```

Comment:
```suggestion
  stepY = (maxY(node)  - minY(node)) / 2;
```
Not sure if it makes a difference, but seems easy enough to ensure we're using the same max and min that we're using for the graph.

Reply:
True this is more concise.